.github/workflows/rebuild-dti-site.yml

```diff
@@ -0,0 +1,12 @@
+name: Rebuild DTI from Master Every 2 Hours
+on:
+  schedule:
+  - cron: '0 */2 * * *'
```

Comment:
This is potentially very wasteful, since netlify build minutes are quite expensive (compare to free github action minutes for open source)... The website is not going to change a lot, and I think one day interval (like the current idol backend syncing bot) is probably good enough.